modules/ossm-performance-scalability.adoc

```diff
@@ -0,0 +1,80 @@
+////
+This module included in the following assemblies:
+- /v2x/customizing-installation-ossm.adoc
+////
+
+[id="ossm-performance-scalability_{context}"]
+= Performance and scalability
+
+{ProductName} strives to provide features with minimal resource overhead and aims to support very large meshes with high request rates while adding minimal latency.
+
+== Performance summary for Istio 1.6.8
+
+The upstream Istio community load tests mesh consists of *1000* services and *2000* sidecars with 70,000 mesh-wide requests per second.
+Running the tests using Istio 1.6.8, generated the following results:
+
+* The Envoy proxy uses *0.5 vCPU* and *50 MB memory* per 1000 requests per second going through the proxy.
+* Istiod uses *1 vCPU* and *1.5 GB* of memory.
+* The Envoy proxy adds *3.12 ms* to the 90th percentile latency.
+* The legacy `istio-telemetry` service (disabled by default in Service Mesh 2.0) uses *0.6 vCPU* per 1000 mesh-wide requests per second for deployments that use Mixer.
+// TODO The Envoy numbers goes down in 1.9, check for the latest data with next version of Istio.
```

Comment:
Yes.  We can publish this content now, but since the performance improves for Istio 1.9, we'll want to remember to update this content when we update the Istio version to 1.9.